services/director-v2/src/simcore_service_director_v2/modules/dask_client.py

```diff
@@ -99,15 +99,15 @@
 class DaskClient:
     app: FastAPI
     dask_subsystem: DaskSubSystem
-    settings: DaskSchedulerSettings
+    settings: DaskComputationalBackendSettings
 
     _subscribed_tasks: List[asyncio.Task] = field(default_factory=list)
 
     @classmethod
     async def create(
         cls,
         app: FastAPI,
-        settings: DaskSchedulerSettings,
+        settings: DaskComputationalBackendSettings,
```

Comment:
Following the discussion above, here we have the ``app`` so it is easy to do instead a property
```python

@dataclass
class DaskClient:
    app: FastAPI
    dask_subsystem: DaskSubSystem

    @property
    def settings(self) -> DaskComputationalBackendSettings:
      return self.app.state.DIRECTOR_V2_DASK_SCHEDULER

```

Reply:
the app is gone